communities.json

```diff
@@ -771,6 +771,13 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "stitches.svg",
+      "title": "Stitches",
+      "quote": "We use Discord to ensure we're building the right tool. We engage with our community, chat about API ideas, offer support, collect feedback and give out early access. It's amazing!",
+      "inviteCode": "H4eG3Mk",
```

Comment:
Did you mean for this to go to the "Modulz" server?

Reply:
Hey @jamiebuilds The way we're using Discord at Modulz is that we have one server and different categories for our sub-products (Stitches and Radix). 

The invite I added here is specifically to the welcome channel Stitches category.

Is that an issue? I was thinking about creating different servers for different sub-products, but thought this would be easier to manage.

Reply:
That's fine, can you double check that this invite code does not have an expiration date tied to it?

Reply:
@jamiebuilds Checked, no expiration date 👍 Thanks!

![image](https://user-images.githubusercontent.com/372831/107429180-f39b4a00-6b23-11eb-9c46-e63b4506c449.png)